users/views.py

```diff
@@ -202,3 +208,17 @@ def update_organization_user(sender, instance: OrganizationUser, **kwargs):
     async_to_sync(channel_layer.group_send)(
         user_group_name, {"type": "send_user", "data": user_data}
     )
+
+
+@api_view(["POST"])
+def retrieve_analytics_app_access_token(request):
+    """ Makes a client credentials request to Auth0 app to get an access token. """
+    url = AUTH0_TOKEN_URL
+    payload = {
+        "grant_type": "client_credentials",
+        "client_id": AUTH0_CLIENT_ID,
+        "client_secret": AUTH0_CLIENT_SECRET,
+        "audience": AUTH0_AUDIENCE,
+    }
+    response = requests.post(url, data=payload)
+    return Response(response.json(), status=status.HTTP_200_OK)
```

Comment:
shouldn't we also check for error here?
And if it is successful, we don't need to pass the status right?

Reply:
I think if the request fails, it will anyway result in an error. And that's how it should be. We can target handling the error message separately.

I believe we do need the 200 status as a parameter for returning response from rest framework.